modules/nix-darwin-unmanaged.nix

```diff
@@ -0,0 +1,75 @@
+{ lib, ... }: {
+  # Disable some modules that conflict with determinate-nixd.
+  disabledModules = [
+    # Wants to add an outdated Nix to the environment, manage nix.conf, manage the
+    # NIX_PATH, manage the Nix build users and group
+    "nix"
+
+    # Wants to configure various Nix settings that want to write to nix.conf
+    "nix/linux-builder.nix"
+
+    # Wants to configure NIX_PATH
+    "nix/nixpkgs-flake.nix"
+
+    # Wants to add to nix.conf
+    "services/hercules-ci-agent"
+
+    # Wants to configure the nix-daemon launchd unit, but determinate-nixd has its own
+    "services/nix-daemon.nix"
+  ];
+
+  ### Setup bogus options so that some things still work as expected.
+
+  # determinate-nixd manages the Nix daemon.
+  options.nix.useDaemon = lib.mkOption {
+    type = lib.types.bool;
+    default = true;
+    internal = true;
+  };
+
+  # Necessary for darwin-rebuild, etc. tools.
+  # https://github.com/LnL7/nix-darwin/blob/55d07816a0944f06a9df5ef174999a72fa4060c7/pkgs/nix-tools/default.nix#L8
+  options.nix.package = lib.mkOption {
+    type = lib.types.package;
+    default = {
+      type = "derivation";
+      outPath = "/nix/var/nix/profiles/default";
+    };
+    internal = true;
+  };
```

Comment:
home-manager needs this to look like a package type or it errors out

Reply:
Sounds like a good comment to live in-code :)